Core/include/Acts/TrackFinding/MeasurementSelector.hpp

```diff
@@ -184,6 +187,28 @@ class MeasurementSelector {
   }
 
  private:
+  template <typename CutType>
+  static CutType VariableCut(
```

Comment:
The idea is that this routine could also add other variable cuts, eg. in pT bins. That's why it is passed `selector`, rather than `selector->etaBins`. Such a change would still need changes to `MeasurementSelector::Config`, but not any of its uses. This might be a moot point if the same routine can be used elsewhere and I move it to a Utility function.
It might be clearer if this were a static member of `MeasurementSelector::Config`, but I think it's clearer if we keep that unambiguously POD (if that's the right term for something with non-POD members).

Reply:
Ah right, so you mean that even though it specifically looks into `selector->etaBins` right now, it might be extended to also look into other members of `selector`?

Reply:
Yes, exactly :)

Reply:
Ok.